content/cds_hooks.md

```diff
@@ -0,0 +1,111 @@
+The CDS Hooks (Clinical Decision Support Hooks) workflow defines a specification for external CDS Service developers to send notification cards to an electronic health record (EHR). These cards are triggered based on conditions such as the opening of a patient chart of a specified demographic(age, gender, etc.) or when a order is selected. These cards can contain information, a suggestion for the user, or a link to launch a SMART app (See [SMART](./smart) for more information).
+
+### Authorization Model ###
+1. Cerner must validate and register every CDS service.
+2. Each client decides whether they want to allow a CDS Service on a domain-by-domain basis.
+3. Each client decides which users and roles can access the CDS Service.
+
+### Supported Hooks ###
+Currently, Cerner supports the following CDS Hooks:
+
+- [Patient View](https://cds-hooks.org/hooks/patient-view/)
+
+## CDS Service Registration ##
+In order to submit a service to be registered, the following information should be provided:
+
+Name         | Required     | Description | Example
+-------------|--------------|-------------|--------
+hook          | Yes | The hook this service should be invoked on. See [Supported Hooks](#supported-hooks). | patient-view
+title         | Yes | The human-friendly name of this service.    | Severe Condition Alerter
+fhir server   | Yes | The FHIR server that the service queries.   | https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d 
+service url   | Yes | The url for the service that the CDS server will call to get the service card response.  | https://company.myservice.com
+description   | No  | Further information about the service. | Sends alerts for patients above age 65 who may have a severe condition.
+owner email   | No  | Required when scopes are provided. The email address of the owner of the service.        | john.smith@company.com
+tenants       | No  | A list of tenants where the service is enabled. | C1941 - ec2458f2-1e24-41c8-b71b-0e701af7583d, A1234 - as1558f2-1e24-44c8-b71b-0e701af787f3d
+fhir scopes   | No  | A list of the FHIR scopes required for any FHIR calls this CDS service makes. | patient/Patient.read, patient/Encounter.read, patient/Observations.read
+allowed links | No  | A list of all links allowed in the response. Any links not in this list will be removed. | https://www.google.com, https://www.cerner.com
+smart apps    | No  | A list of launch urls for any SMART applications that may be linked to in CDS service responses. | https://www.myapp.com, https://www.otherapp.org
+
+For more information on CDS Service attributes see the Discovery Response section [here](https://cds-hooks.hl7.org/1.0/#response).
+
+## CDS Service Workflow ##
+When the condition in the EHR is met, the CDS Hooks Server will make a GET request to the provided service url. The service should returns an array of cards formatted according to the [CDS Service HTTP Response specification](https://cds-hooks.hl7.org/1.0/#http-response). 
```

Comment:
nit: 'return' instead of 'returns'

Reply:
https://github.com/cerner/fhir.cerner.com/pull/707/commits/f6d822ea6e714ca5ba3db1d5e97b464eec9b9c68#diff-179aa7a92baef1c89bbd6fabd30f690e85b09c4e22896697d9d13b17dc80d5e7R61